docs/excel/custom-functions-architecture.md

```diff
@@ -33,12 +33,12 @@ The browser engine runtime supports the Office.js APIs. Keep in mind that any of
 
 Your custom functions code cannot directly interact with code in other parts of your web add-in, like the task pane because they are in different runtimes. But in some scenarios you may need to share data, such as passing a token.
 
-`AsyncStorage` can be used to store data from your custom functions and get data from your task pane code. For more information about storing and sharing data, see [Saving and sharing state](custom-functions-overview.md#saving-and-sharing-state).
+`Office.Storage` can be used to store data from your custom functions and get data from your task pane code. For more information about storing and sharing data, see [Save and share state](custom-functions-save-state.md).
```

Comment:
```suggestion
The `Office.Storage` object can be used to store data from your custom functions and get data from your task pane code. For more information about storing and sharing data, see [Save and share state](custom-functions-save-state.md).
```

Reply:
I like this being a first usage "Office.Storage object", but I think we can omit the namespace later to just "Storage object".